xbump

```diff
@@ -29,7 +29,14 @@ dirs="$dirs $(./xbps-src show "$pkg" |
 git add $dirs
 
 if git diff --quiet --cached --diff-filter=A -- srcpkgs/"$pkg"/template; then
-	msg="$pkg: update to $version."
+	msg="$pkg: update to $version"
```

Comment:
But muh periods??!

Reply:
`msg+="."` at the end then

Reply:
otherwise the message would look `abc: update to 0.1.2., orphan`. abby's suggestions makes sense here.